torch/csrc/jit/codegen/cuda/parser.cpp

```diff
@@ -1514,6 +1514,36 @@ class IrParser {
       }
     }
 
+    {
```

Comment:
Please pull the frontend change into a separate PR.

Reply:
Done.
debug_lower_GPU PR: https://github.com/azazhu/pytorch/tree/fw/dump_debug
frontend PR: https://github.com/azazhu/pytorch/tree/fw/idx_sel_frontend
backend PR: https://github.com/azazhu/pytorch/tree/fw/idx_sel_backend